axlearn/experiments/text/gpt/common.py

```diff
@@ -636,6 +637,7 @@ def get_trainer_config_fn(
     keep_every_n_steps: int = 50_000,
     save_every_n_steps: Optional[int] = None,
     init_state_builder: Optional[state_builder.Builder.Config] = None,
+    logical_feed_indices: Optional[Sequence[int]] = None,
```

Comment:
Do we need this? I think InputDispatcher can figure it out automatically.

Reply:
I think it's still needed when per host bs < 1. In other cases, it's optional.

Reply:
See https://github.com/apple/axlearn/blob/fad264b2b2d60746585cdbc46089528ee037fb56/axlearn/common/input_dispatch.py#L62-L64.

Reply:
Done. Then it looks like a lot of our internal models have this redundantly specified.